src/main/java/nextstep/subway/dto/LineRequest.java

```diff
@@ -0,0 +1,44 @@
+package nextstep.subway.dto;
+
+public class LineRequest {
+
+    private String name;
+    private String color;
+    private Long upStationId;
+    private Long downStationId;
+    private Long distance;
+
+    public String getName() {
+        return name;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public Long getUpStationId() {
+        return upStationId;
+    }
+
+    public Long getDownStationId() {
+        return downStationId;
+    }
+
+    public Long getDistance() {
+        return distance;
+    }
+
+    private LineRequest(String name, String color, Long upStationId, Long downStationId,
```

Comment:
```suggestion
    private LineRequest(
    String name, 
    String color, 
    Long upStationId, 
    Long downStationId,
    Long distance
```
아마 코드 템플릿에서 정한 길이가 있어서 자동으로 개행이 되는 것 같습니다만, 파라미터 한개만 개행되는 것보다는 이렇게 개행해주는게 보기 편하지 않을까요...?

Reply:
넵 코드템플릿을 맞추다 보니 보기 찝찝해도 그냥 넘어간 부분도 있었는데...
가독성 > 코드 템플릿을 우선순위로 해야할것같아요 아니면 템플릿설정을 좀 변경해보아야겠습니다 ㅠㅠ